string rob_name
bool rob_status

string task_name
bool task_status
float32 task_x
float32 task_y
float32 task_theta
